src/Relude/Extra/Foldable1.hs

```diff
@@ -0,0 +1,154 @@
+{-# LANGUAGE InstanceSigs     #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Relude.Extra.Foldable1
```

Comment:
Okay, we actually didn't invent `Foldable1`. So we need to write copyright notice at top of this module and give credits to the initial authors of `Foldable1`.

Check this module to see how we give credits:

* https://github.com/kowainik/relude/blob/9c26c536758d8b8573c4cb0606c25bd311e7565f/src/Relude/Applicative.hs#L3-L13

And see header of `Foldable1` module from `semigroupoids` package to see who is original author of the idea and who should go before `Kowainik`:

* http://hackage.haskell.org/package/semigroupoids-5.3.1/docs/src/Data.Semigroup.Foldable.Class.html